libraries/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -286,27 +285,37 @@ static IotMqttError_t _getIncomingPacket( void * pNetworkConnection,
                          pMqttConnection,
                          ( unsigned long ) pIncomingPacket->remainingLength,
                          ( unsigned long ) pIncomingPacket->type );
```

Comment:
This log seems to have 2 copies and the above doesn't seems at the right place. You may want to add the coverity annotations to the right one below